src/nxdoc/nuxeo-server/installation/nuxeo-storage-alternatives.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Nuxeo Storage Alternatives
+is_overview: true
+review:
+    comment: ''
+    date: '2021-03-10'
+    status: ok
+toc: false
+tree_item_index: 1390
+---
+
+Nuxeo is pluggable so that it can be adapted to different deployment environments and use cases.
+
+This means you can define where you want to manage your data, and because the answer may depend on the type of data. Nuxeo provides different types of backends for different types of storage.
+
+## Document Storage
+
+You can configure:
+
+- Where you store the document metadata and hierarchy
+    - SQL Database (PostgresSQL, Oracle, MSSQL, MySQL, Amazon RDS)
+    - MongoDB
+- Where you store the binary streams (the files you attach to documents)
+    - Simple FileSystem
+    - SQL Database
```

Comment:
Not sure we have an implementation to store blobs in SQL, but we do have one for MongoDB (GridFS).